### PR TITLE
[Clang][OHOS] Keep ARM ABI selection logic in sync between Clang and LLVM

### PR DESCRIPTION
**Pull request by brad0**

### CODE REVIEW

**Comment by brad0** [comment]
ping.

File: llvm/lib/TargetParser/ARMTargetParser.cpp [hunk 1]
```diff
@@ -519,6 +519,7 @@ StringRef ARM::computeDefaultTargetABI(const Triple &TT, StringRef CPU) {
   case Triple::GNUEABIHF:
   case Triple::MuslEABI:
   case Triple::MuslEABIHF:
+  case llvm::Triple::OpenHOS:
```
Comment (asl) [inline llvm/lib/TargetParser/ARMTargetParser.cpp:522]:
```suggestion
  case Triple::OpenHOS:
```

Comment (brad0) [reply llvm/lib/TargetParser/ARMTargetParser.cpp]: Oops, my bad. That has been corrected.

**Comment by brad0** [comment]
Ping.

**Comment by kpdev** [comment]
@brad0 Thank you for the patch, and sorry for late reply - there were wrong settings with notifications, won't happen again

**Pull request merged**